schema.asn

```diff
@@ -34,27 +34,27 @@ Configuration-Structure-Schema DEFINITIONS AUTOMATIC TAGS ::= BEGIN
    FlexibleRange4 ::= SEQUENCE {
       -- The count of single sequence [1..4] which is
       -- a 2-bit count
-      single SEQUENCE SIZE(1..4) OF Integer4 OPTIONAL,
+      single SEQUENCE SIZE(1..4) OF Integer2 OPTIONAL,
       -- The intenger range is [0..3]
-      range SEQUENCE OF Range4 OPTIONAL
+      range SEQUENCE OF Range2 OPTIONAL
    }
 
    -- 3-bit flexible range [0..7]
    FlexibleRange8 ::= SEQUENCE {
       -- The count of single sequence [1..8] which is 
       -- a 3-bit count
-      single SEQUENCE SIZE(1..8) OF Integer8 OPTIONAL,
+      single SEQUENCE SIZE(1..8) OF Integer3 OPTIONAL,
       -- The intenger range is [0..7]
-      range SEQUENCE OF Range8 OPTIONAL
+      range SEQUENCE OF Range3 OPTIONAL
    }
 
    -- 4-bit flexible range [0..15]
    FlexibleRange16 ::= SEQUENCE {
       -- The count of single sequence [1..16] which is
       -- a 4-bit count
-      single SEQUENCE SIZE(1..16) OF Integer8 OPTIONAL,
+      single SEQUENCE SIZE(1..16) OF Integer3 OPTIONAL,
       -- The intenger range is [0..15]
-      range SEQUENCE OF Range16 OPTIONAL
+      range SEQUENCE OF Range4 OPTIONAL
```

Comment:
also change FlexibleRangeN to width base? FlexibleRange2, 3, 4?

Reply:
Good point. Will do.